me.gladwell.eclipse.m2e.android.test/src/me/gladwell/eclipse/m2e/android/test/AndroidMavenPluginTestCase.java

```diff
@@ -219,7 +221,7 @@ protected boolean classpathContainerContains(IJavaProject project, String id, St
         try {
             return getClasspathEntry(project, id, path) != null;
         } catch (Exception e) {
-            e.printStackTrace();
+            debug("entry was not found", e);
```

Comment:
I'm not sure we should hide this log messages for tests. Better would be to refactor `getClasspathEntry` so it doesn't throw an exception to indicate an entry was not found.